tests/behat/features/bootstrap/SocialMinkContext.php

```diff
@@ -317,4 +317,17 @@ public function iShouldSeeTheTextCertainNumberTimes($text, $expectedNumber): voi
       throw new \Exception('Found '.$numberText.' times of "'.$text.'" when expecting '.$expectedNumber);
     }
   }
+
+  /**
+   * @When /^I hover over the element "([^"]*)"(?:|.*)$/
+   */
+  public function iHoverOverTheElement($locator) {
+    $session = $this->getSession();
+    $element = $session->getPage()->find('css', $locator);
+
+    if (null === $element) {
+      throw new \InvalidArgumentException(sprintf('Could not evaluate CSS selector: "%s"', $locator));
```

Comment:
We can better handle this exception if we use a ElementNotFound exception.

Reply:
Fixed